src/cljs/org/broadinstitute/firecloud_ui/common/style.cljs

```diff
@@ -114,7 +114,12 @@
    text])
 
 (defn render-entity [namespace name snapshot-id]
+  (render-name-id (str namespace "/" name) snapshot-id))
```

Comment:
bah - this adds a warning when compiling:
```
Use of undeclared Var org.broadinstitute.firecloud-ui.common.style/render-name-id
```
I think all you have to do is move `render-entity` beneath `render-name-id`, so that `render-name-id` is defined by the time `render-entity` is compiled.